tests/ui/test_ui.py

```diff
@@ -225,58 +226,107 @@ def just_set_message_view(self: Any) -> None:
         frame.assert_called_once_with(
             view.body, col(), focus_part="body", footer=footer_view()
         )
+        assert view.frame == frame()
+        show_left_panel.assert_called_once_with(visible=True)
 
     @pytest.mark.parametrize("autohide", [True, False])
-    @pytest.mark.parametrize("visible, width", [(True, LEFT_WIDTH), (False, TAB_WIDTH)])
+    @pytest.mark.parametrize("visible", [True, False])
+    @pytest.mark.parametrize(
+        "expected_overlay_options",
+        [
+            (
+                "left",
+                None,
+                "given",
+                LEFT_WIDTH + 1,
+                None,
+                0,
+                0,
+                "top",
+                None,
+                "relative",
+                100,
+                None,
+                0,
+                0,
```

Comment:
There are two points I'd make here:
* While we do have them elsewhere, having one value parametrized has little point and can be misleading - better to have the value "in" the test itself
* This big list of values takes up lots of space and is (somewhat) shared with the other test - can we combine them?